fenix/app/src/main/res/values/strings.xml

```diff
@@ -306,12 +306,10 @@
     <string name="onboarding_home_enable_notifications_negative_button">Not now</string>
 
     <!-- Juno first user onboarding flow experiment, strings are marked unused as they are only referenced by Nimbus experiments. -->
-    <!-- Title for set firefox as default browser screen used by Nimbus experiments. Nimbus experiments do not support string placeholders.
-        Note: The word "Firefox" should NOT be translated -->
-    <string name="juno_onboarding_default_browser_title_nimbus" tools:ignore="UnusedResources">Make Firefox your go-to browser</string>
-    <!-- Description for set firefox as default browser screen used by Nimbus experiments. Nimbus experiments do not support string placeholders.
-        Note: The word "Firefox" should NOT be translated -->
-    <string name="juno_onboarding_default_browser_description_nimbus" tools:ignore="UnusedResources">Firefox puts people over profits and defends your privacy by blocking cross-site trackers.\n\nLearn more in our privacy notice.</string>
+    <!-- Title for set firefox as default browser screen used by Nimbus experiments. -->
+    <string name="juno_onboarding_default_browser_title_nimbus" tools:ignore="UnusedResources">We love keeping you safe</string>
```

Comment:
This string landed back in April so it would've been exposed to localization by now. So, we will need to use our normal string removal process and introduce a `_2` https://github.com/mozilla-mobile/firefox-android/blob/main/fenix/docs/Working-with-Strings.md